src/server/database/heuristics.js

```diff
@@ -8,9 +8,21 @@
 */
 
 // --------- Fake Heuristics ---------------
-function run(layouts, callback) {
+function run(layouts, numEntries, callback) {
     let layout = layouts.toArray()
       .then((result) => {
+
+        //Return n most recent layouts
+        if (numEntries && result.length <= 1){
+          return [];
+        }
+        if(numEntries && result.length < -10){
```

Comment:
I think there is something wrong here.

Reply:
@d2fong  Ah, forgot to change the comparisons. They are all set to numEntries which is non-negative, now.